### PR TITLE
InfiniBand Performance Tests

### PR DESCRIPTION
**Pull request by schlad**
WIP: PoC for IB IPoIB Performance automated tests

As I have not much of the actual experience with the openQA I would like to get some reviews of this PR which is about checking the performance of InfiniBand

### CODE REVIEW

File: tests/kernel/ib_perf_tests.pm [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+    barrier_wait('IBTEST_DONE');
+}
+
+sub ibtest_master {
+    my $master             = get_required_var('IBTEST_IP1');
+    my $slave              = get_required_var('IBTEST_IP2');
+
+    # do all test preparations and setup
+    zypper_call('--gpg-auto-import-keys ref');
+    zypper_call('in perftest');
```
Comment (frankenmichl) [inline tests/kernel/ib_perf_tests.pm]: As you need to intall the same dependencies for master and slave, you should move this to run()

File: tests/kernel/ib_perf_tests.pm [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+    barrier_wait('IBTEST_BEGIN');
+    script_run("ib_write_bw $slave > /tmp/ib_write_bw.txt", 10);
+    upload_logs("/tmp/ib_write_bw.txt");
```
Comment (frankenmichl) [inline tests/kernel/ib_perf_tests.pm]: You should use assert_script_run() here or check the log uploaded for results in order to have any chance the test could fail.

File: tests/kernel/ib_tests.pm [hunk 3]
```diff
@@ -9,7 +9,7 @@
 
 # Summary: run InfiniBand test suite hpc-testing
 #
-# Maintainer: Michael Moese <mmoese@suse.de, Nick Singer <nsinger@suse.de>
+# Maintainer: Michael Moese <mmoese@suse.de>, Nick Singer <nsinger@suse.de>
```
Comment (frankenmichl) [inline tests/kernel/ib_tests.pm]: While this is correct, this should not be part of this PR

**Comment by OleksandrOrlov** [comment]
@schlad Hi! Do you plan to look at this ever again?

**Comment by schlad** [comment]
> @schlad Hi! Do you plan to look at this ever again?

yes

**Comment by schlad** [comment]
> @schlad Hi! Do you plan to look at this ever again?

hell with this. I will never find time for this PR. So I will close it for now and redo it while I have some time finally.

**Pull request closed without merging**